### DISCUSSION TITLE
How to add pipenv to the PATH environment variable?

### DISCUSSION THREAD

**Discussion by Ankitsgithubs** [Atri-Labs/atrilabs-engine#586] [Help: Installation & Start] [answered]
1. Click on the Search bar and type "environment variable".
2. Click on  "Edit the system environment variables".
3. Click on the "Environment Variables" button.
4. In the "User variables for YOUR_USER_NAME" section, select the "Path" variable and click "Edit".
5. Click on "New" and then click "Browse".
6. browse the path which would look like this "C:\Users\YOUR_USER\AppData\Local\Programs\Python\Python310".

If you are unable to find the path like the above one then it must be hidden. You have select show hidden items from the settings. After this, you will be able to view "AppData".

7. After finding AppData, go to python310(check the point no 6 .  where it is located) and then select the "script" folder  in python310 folder.  Now select the whole path. And paste it in point no.5.
8. After point no.7 close the terminal and reopen it and follow the instruction in this document [click here](https://docs.atrilabs.com/getting-started/installation/install_with_pipenv)

**Comment by cruxcode** [comment, marked as answer]
Based on the python version installed on your computer, you might have a different folder than Python310. To check your python version run `python3 --version`.

**Comment by Bimal194** [comment]
After all these steps , when I write `pipenv install atri`
It gives following error:

Traceback (most recent call last):
  File "c:\users\hp\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\hp\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals) ,,,,,,,,,,,,,,,
 File "c:\users\hp\appdata\local\programs\python\python39\lib\pathlib.py", line 1221, in stat
    return self._accessor.stat(self)
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Users\\hp\\AppData\\Local\\Programs\\Hyper\\resources\\binC:\\Users\\hp\\AppData\\Roaming\\Microsoft\\Windows\\Start Menu\\Programs\\GitHub, Inc'
